storage/rocksdb/rdb_utils.h

```diff
@@ -219,6 +219,16 @@ inline int purge_all_jemalloc_arenas() {
 #endif
 }
 
+template <typename... Params>
+// format attribute would be great here too, but it's unavailable with the
+// variadic templates
+MY_ATTRIBUTE((cold, noreturn, noinline)) void fatal_error(const char *fmt,
```

Comment:
Can you rename this to rdb_fatal_error()?

Reply:
Done

(the reason I did not add `rdb_` prefix before was that I thought namespace `myrocks::` did its job)